src/views/maps/mapsets.twig

```diff
@@ -44,7 +44,16 @@
                             {% else %}
                                 <div class="qadge mapset-7k">7K</div>
                             {% endif %}
-                            {% if map.ranked_status == 2 %}
+                            {% if map.ranking_status == 0 %}
+                                <div class="qadge status-pending">Pending</div>
+							{# Denied = 1 -> Back to unranked #}
+                            {% elseif map.ranking_status == 2 %}
+                                <div class="qadge status-blacklisted">Blacklisted</div>
+                            {% elseif map.ranking_status == 3 %}
+                                <div class="qadge status-onhold">OnHold</div>
```

Comment:
probably separate the words

Reply:
I wondered whether to keep separate or not, but afaik it's written like this on ranking queue ? will check when i can

Reply:
There is a space, omw to add it, thanks !